production/ksonnet/loki/index-gateway.libsonnet

```diff
@@ -0,0 +1,66 @@
+{
+  _config+:: {
+      use_index_gateway: false,
+      index_gateway_pvc_size: '10Gi',
```

Comment:
Should we default to something bigger? 100Gi maybe?

Reply:
For a small cluster like dev clusters in our case, it would be too much. Most users won't need that much space if they do not set query readiness, which defaults to 0. If at all, we can go for `50Gi` for now?

Reply:
This is a tricky one, what do we default the querier pvcs too?

Reply:
I think if someone is going to use the index gateway it would likely be because they run a bigger cluster, so I think 50G or bigger as a default would be better

Reply:
We default querier pvcs to 10Gi. I have set it to 50Gi for index gateway.
Please let me know if you want to increase it further.